ethereum/executionclient/src/integration-test/java/tech/pegasys/teku/ethereum/executionclient/rest/RestBuilderClientTest.java

```diff
@@ -673,9 +673,12 @@ private void runGetPayloadSuccessAsJson() {
               verifyBuilderPayloadResponse(builderPayload);
             });
     final Consumer<RecordedRequest> containsConsensusVersionHeader =
-        req ->
-            assertThat(req.getHeader("Eth-Consensus-Version"))
-                .isEqualTo(milestone.name().toLowerCase(Locale.ROOT));
+        req -> {
+          assertThat(req.getHeader("Eth-Consensus-Version"))
```

Comment:
We'd better use the `HEADER_CONSENSUS_VERSION` constant btw

Reply:
well... I think in test we should testing the strings cose if we mess up the constants we will pass the tests :)

Reply:
oh my bad that's right!